lisa/sut_orchestrator/azure/common.py

```diff
@@ -2274,13 +2274,27 @@ def get_primary_ip_addresses(
         nic_name = get_matched_str(network_interface.id, NIC_NAME_PATTERN)
         nic = network_client.network_interfaces.get(resource_group_name, nic_name)
         if nic.primary:
+            # Check if IPv6 is actually enabled on the NIC
+            # if its enabled, use it as internal ip.
+            ipv6_enabled = any(
```

Comment:
What's the difference between this value and `use_ipv6`?

Reply:
Ack, let me rework the PR